gym_roboy/envs/ros_proxy.py

```diff
@@ -133,13 +131,14 @@ def forward_new_goal(self, goal_joint_angle):
         self.sphere_axis1.publish(msg1)
         self.sphere_axis2.publish(msg2)
 
-    def set_new_goal(self):
+    def get_new_goal_joint_angles(self):
         while self._check_service(self.goal_client):
             req = GymGoal.Request()
             future = self.goal_client.call_async(req)
             rclpy.spin_until_future_complete(self.node, future)
             res = future.result()
             if res is not None:
                 self.node.get_logger().info("feasible: " + str(res.q))
+            self._forward_new_goal(res.q)
```

Comment:
It always shows the goal at the origin this way don't know why.

Reply:
Is the line above printing the origin as well?

Reply:
put it inside if now works